src/style.css

```
@@ -0,0 +1,69 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  position: relative;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  list-style-type: none;
+  height: 60px;
+  padding: 0;
+}
+
+.list_item {
+  font-family: inherit;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-left: 20px;
+  cursor: pointer;
+}
+
+.list_item:first-child {
+  margin-left: 0;
+}
+
+.link_item {
+  color: #000;
+  padding: 23px 0;
+  line-height: 14px;
+}
+
+.is-active,
+.link_item:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  width: 36px;
```

```suggestion
  width: 100%;
```
якщо текст елемента стане довшим то ширина 37 пікселів уже не підійде, а якщо написати 100% то вона змінюватиметься разом з елементом